docs/compute/decentralized_live_migration.md

```diff
@@ -0,0 +1,248 @@
+
+### Source migration resource
+
+The source migration resource now includes an additional field with two sub-fields:
+
+1. `migrationID` (identical to the target migration resource) 
+2. `connectURL`. This connection URL is used to connect to the synchronization controller on the target cluster. 
+
+There are two methods to acquire the synchronization controller's address:
+
+1. **Accessing the KubeVirt CR on the Target Cluster:** If you have access to the KubeVirt CR, a new field named `synchronizationAddresses` will provide a list of address/port combinations for the `connectURL`.  
+2. **After Target Migration Resource Creation:** If you lack KubeVirt CR access, once the target migration resource is created, its status will contain a `synchronizationAddresses` field with the same list of address/port combinations.
+
+The key distinction between these options is that Method 1 allows you to obtain the connection URL in advance, enabling the creation of the source migration resource simultaneously with or prior to the target migration resource.
+
+Once the connection URL is acquired, its value should be placed in the `connectURL` field of the source migration resource.
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstanceMigration
+metadata:
+  name: example-source-migration
+spec:
+  sendTo:
+	connectURL: "<synchronization address>"
+	migrationID: <unique identifier, same as target migration resource migrationID>
+  vmiName: <name of the source vmi>
+```
+
+### Configuring KubeVirt CA with cross cluster live migration
+
+When migrating between two clusters, each cluster will have its own installation of KubeVirt. Each KubeVirt installation will create and use its own Certificate Authority (CA). This CA is used to secure communication between the synchronization controllers and the virt-handlers during migration. For each cluster to trust the other cluster, the public CA keys must be exchanged between the clusters.
+
+The public CA key is stored in a `ConfigMap` in the kubevirt namespace. This `ConfigMap` is named `kubevirt-ca`. The following is an example of this `ConfigMap`:
+
+```yaml
+apiVersion: v1
+data:
+  ca-bundle: |
+    -----BEGIN CERTIFICATE-----
+...
+    -----END CERTIFICATE-----
+    -----BEGIN CERTIFICATE-----
+...
+    -----END CERTIFICATE-----
+kind: ConfigMap
+metadata:
+  name: kubevirt-ca
+  namespace: kubevirt
+```
+
+In the same namespace, there will be another `ConfigMap` named `kubevirt-external-ca` which will have a `ca-bundle` field as part of its data. In order for cluster1 to trust the CA from cluster2, take the ca-bundle from cluster2's `kubevirt-ca` `ConfigMap` and copy the `ca-bundle` into cluster1's `kubevirt-external-ca` and vice versa. After you update the `kubevirt-external-ca`, it will automatically clear out the contents of the `kubevirt-external-ca` but copy the contents into the `kubevirt-ca` `ConfigMap`. It will remove invalid, duplicate and expired keys before doing so.
```

Comment:
```suggestion
In the same namespace, there will be another `ConfigMap` named `kubevirt-external-ca` which will have a `ca-bundle` field as part of its data. In order for Cluster 1 to trust the CA from Cluster 2, take the ca-bundle from Cluster 2's `kubevirt-ca` `ConfigMap` and copy the `ca-bundle` into Cluster 1's `kubevirt-external-ca` and vice versa. After you update the `kubevirt-external-ca`, it will automatically clear out the contents of the `kubevirt-external-ca` but copy the contents into the `kubevirt-ca` `ConfigMap`. It will remove invalid, duplicate and expired keys before doing so.
```
To match diagram and be consistent with earlier name